components/esp_modem/examples/simple_cmux_client/components/SIM7070_gnss/nmea_parser.h

```diff
@@ -0,0 +1,137 @@
+    GPS_MODE_2D,          /*!< 2D GPS */
+    GPS_MODE_3D           /*!< 3D GPS */
+} gps_fix_mode_t;
+
+/**
+ * @brief GPS satellite information
+ *
+ */
+typedef struct {
+    uint8_t num;       /*!< Satellite number */
+    uint8_t elevation; /*!< Satellite elevation */
+    uint16_t azimuth;  /*!< Satellite azimuth */
+    uint8_t snr;       /*!< Satellite signal noise ratio */
+} gps_satellite_t;
+
+/**
+ * @brief GPS time
+ *
+ */
+typedef struct {
+    uint8_t hour;      /*!< Hour */
+    uint8_t minute;    /*!< Minute */
+    uint8_t second;    /*!< Second */
+    uint16_t thousand; /*!< Thousand */
+} gps_time_t;
+
+/**
+ * @brief GPS date
+ *
+ */
+typedef struct {
+    uint8_t day;   /*!< Day (start from 1) */
+    uint8_t month; /*!< Month (start from 1) */
+    uint16_t year; /*!< Year (start from 2000) */
+} gps_date_t;
+
+/**
+ * @brief NMEA Statement
+ *
+ */
+typedef enum {
+    STATEMENT_UNKNOWN = 0, /*!< Unknown statement */
+    STATEMENT_GGA,         /*!< GGA */
+    STATEMENT_GSA,         /*!< GSA */
+    STATEMENT_RMC,         /*!< RMC */
+    STATEMENT_GSV,         /*!< GSV */
+    STATEMENT_GLL,         /*!< GLL */
+    STATEMENT_VTG          /*!< VTG */
+} nmea_statement_t;
+
+/**
+ * @brief GPS object
+ *
+ */
+struct gps_s {
+    float latitude;                                                /*!< Latitude (degrees) */
+    float longitude;                                               /*!< Longitude (degrees) */
+    float altitude;                                                /*!< Altitude (meters) */
+    gps_run_t run;                                                 /*!< run status */
+    gps_fix_t fix;                                                 /*!< Fix status */
+    uint8_t sats_in_use;                                           /*!< Number of satellites in use */
+    gps_time_t tim;                                                /*!< time in UTC */
+    gps_fix_mode_t fix_mode;                                       /*!< Fix mode */
+    float dop_h;                                                   /*!< Horizontal dilution of precision */
+    float dop_p;                                                   /*!< Position dilution of precision  */
+    float dop_v;                                                   /*!< Vertical dilution of precision  */
+    uint8_t sats_in_view;                                          /*!< Number of satellites in view */
+    gps_date_t date;                                               /*!< Fix date */
+    float speed;                                                   /*!< Ground speed, unit: m/s */
+    float cog;                                                     /*!< Course over ground */
+    float hpa;                                                     /*!< Horizontal Position Accuracy  */
+    float vpa;                                                     /*!< Vertical Position Accuracy  */
+};
+
+typedef struct gps_s gps_t;
```

Comment:
Could we remove one of these? It can be confusing to have two different alias for the same type.